v1/pipeline/steps/cd/k8s-rolling-step-info.yaml

```diff
@@ -3,56 +3,20 @@ allOf:
 - $ref: ../../common/step-spec-type.yaml
```

Comment:
there is one field _canaryStepFqn_ lets check that too.

Reply:
This field is not in the Schema. This is the FQN of the step form stage that is being calculated during the plan-creation.